File: @LV1/1.4/heartrate/heartrate.py
n = int(input())

for _ in range(n):
    line = input().split(' ')
    b, p = int(line[0]), float(line[1])

    bpm = 60*b/p
    abpm = 60/p
    print(bpm-abpm, bpm, bpm+abpm)

File: @LV1/1.4/heartrate/heartrate.rb
n = gets.to_i

n.times do
  line = gets.split
  b, p = line[0].to_i, line[1].to_f

  bpm = 60*b/p
  abpm = 60/p
  puts "#{bpm-abpm} #{bpm} #{bpm+abpm}"
end